pkg/reconciler/knativeserving/knativeserving_controller.go

```diff
@@ -181,7 +182,6 @@ func (r *Reconciler) install(manifest *mf.Manifest, instance *servingv1alpha1.Kn
 // Check for all deployments available
 func (r *Reconciler) checkDeployments(manifest *mf.Manifest, instance *servingv1alpha1.KnativeServing) error {
 	r.Logger.Debug("Checking deployments")
-	defer r.updateStatus(instance)
```

Comment:
@jcrossley3 Has this line intentionally been removed?

Reply:
I think it's redundant due to https://github.com/knative/serving-operator/blob/39757e9cc8d2d0c94096180b1ee886e84d5a4455/pkg/reconciler/knativeserving/knativeserving_controller.go#L97